bin/gem2rpm

```diff
@@ -9,7 +9,21 @@ require 'fileutils'
 require 'open-uri'
 require 'uri'
 
-options = Gem2Rpm::Configuration.instance.options
+options = begin
+  Gem2Rpm::Configuration.instance.options
+rescue Gem2Rpm::Configuration::ConfigurationError => e
```

Comment:
I don't like this exception name. The "configuration" is repeated there. Why not go with "InvalidOption" which is the name of the originally handled exception.